go.mod

```diff
@@ -9,40 +9,75 @@ require (
 	github.com/Azure/go-autorest/autorest/azure/auth v0.4.2
 	github.com/Azure/go-autorest/autorest/to v0.3.0
 	github.com/Azure/go-autorest/autorest/validation v0.2.0 // indirect
-	github.com/aws/aws-sdk-go v1.13.12
+	github.com/aws/aws-sdk-go v1.28.2
 	github.com/docker/spdystream v0.0.0-20170912183627-bc6354cbbc29 // indirect
 	github.com/evanphx/json-patch v4.5.0+incompatible
-	github.com/go-ini/ini v1.28.2 // indirect
 	github.com/gobwas/glob v0.2.3
 	github.com/gofrs/uuid v3.2.0+incompatible
 	github.com/golang/protobuf v1.4.2
 	github.com/hashicorp/go-hclog v0.0.0-20180709165350-ff2cf002a8dd
 	github.com/hashicorp/go-plugin v0.0.0-20190610192547-a1bc61569a26
-	github.com/jmespath/go-jmespath v0.0.0-20160202185014-0b12d6b521d8 // indirect
 	github.com/joho/godotenv v1.3.0
-	github.com/kubernetes-csi/external-snapshotter/v2 v2.1.0
+	github.com/kubernetes-csi/external-snapshotter/v2 v2.1.1-0.20200617021606-4800ca72d403
```

Comment:
It may be worth including https://github.com/kubernetes-csi/external-snapshotter/releases/tag/v2.2.0-rc1 instead, now that that's been released

Reply:
I'm not sure about including an RC...curious what @ashish-amarnath thinks.

Reply:
Yeah, I think it's better to avoid a pre-release version, esp if we are not depending on changes in the release.

Reply:
It's less "use RC or normal version" and more "use the currently pinned commit version or the newer RC", because unfortunately it does have a change that this PR needs (upgrading to import k8s 1.18)

Reply:
Got it. In that case, I'd prefer to use the RC.